wherehows-dao/src/main/java/wherehows/models/table/DictDataset.java

```diff
@@ -15,98 +15,99 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.PrePersist;
 import javax.persistence.PreUpdate;
 import javax.persistence.Table;
-import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
 
 @Data
 @Entity
 @Table(name = "dict_dataset")
-@AllArgsConstructor
 @NoArgsConstructor
 public class DictDataset {
 
   @Id
-  int id;
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private int id;
 
-  @Column(name = "name", nullable = false)
-  String name;
+  @Column(name = "\"name\"", nullable = false)
```

Comment:
Why is the double quote needed?

Reply:
This is one bug that wasted my a couple hours. MySQL reserved keywords need be be double quoted to avoid potential SQL error. 
https://stackoverflow.com/questions/2224503/creating-field-with-reserved-word-name-with-jpa

Reply:
Argh I see. Can't believe that JPA isn't handling this automatically...